pricegraph/Cargo.toml

```diff
@@ -11,7 +11,7 @@ harness = false
 [dependencies]
 arbitrary = { version = "0.4", optional = true, features = ["derive"] }
 petgraph = "0.5"
-primitive-types = "0.6.2"
+primitive-types = "0.7"
```

Comment:
Looks like this isn't being upgraded by dependabot.

Reply:
I think this is because it was explicitly downgraded to be compatible with `core.`